packages/frontend/src/__tests__/signupPage.js

```diff
@@ -28,18 +39,19 @@ test('should render SignupPage correctly with content[0]', () => {
         loadVotesForPostId: jest.fn(),
         loadVotesForCommentId: jest.fn(),
         getAirdrop: jest.fn(),
+        signUp: jest.fn(),
+        setSignUpPromise: jest.fn(),
     }
 
-    render(
-        <UserContext.Provider value={userData}>
-            <PostContext.Provider value={postData}>
-                <SignupPage />
-            </PostContext.Provider>
-        </UserContext.Provider>
-    )
+    renderSignupPage(userData, postData)
     expect(
         screen.getByText(
             /currently, UniRep Social is an invite only community. Please enter your invitation code below./i
         )
     ).toBeInTheDocument()
+    expect(screen.getByText(/join us/i)).toBeInTheDocument()
+    // generate text on signup page
+    const textbox = screen.getByRole('textbox')
```

Comment:
Maybe use a variable here

Reply:
Not sure I am understanding where exactly the variable is needed. The `currently UniRep Social...` and `join us` regex are what the page renders and I don't think there's a way to refactor the `textbox` here.

Reply:
Ah sorry i meant the line below. `lorem ipsum dolor sit amet`.

Reply:
Gotcha, will implement changes.